src/resolve/resolved-dns-packet.h

```diff
@@ -361,6 +361,25 @@ const char *format_dns_ede_rcode(int i, char buf[static DECIMAL_STR_MAX(int)]);
 const char* dns_protocol_to_string(DnsProtocol p) _const_;
 DnsProtocol dns_protocol_from_string(const char *s) _pure_;
 
+/* https://www.iana.org/assignments/dns-svcb/dns-svcb.xhtml#dns-svcparamkeys */
+enum {
+        DNS_SVC_PARAM_KEY_MANDATORY = 0, /* RFC 9460, section 8 */
+        DNS_SVC_PARAM_KEY_ALPN = 1, /* RFC 9460 section 7.1 */
+        DNS_SVC_PARAM_KEY_NO_DEFAULT_ALPN = 2, /* RFC 9460, Section 7.1 */
+        DNS_SVC_PARAM_KEY_PORT = 3, /* RFC 9460 section 7.2 */
+        DNS_SVC_PARAM_KEY_IPV4HINT = 4, /* RFC 9460 section 7.3 */
+        DNS_SVC_PARAM_KEY_ECH = 5, /* RFC 9460 */
+        DNS_SVC_PARAM_KEY_IPV6HINT = 6, /* RFC 9460 section 7.3 */
+        DNS_SVC_PARAM_KEY_DOHPATH = 7, /* RFC 9461 */
+        DNS_SVC_PARAM_KEY_OHTTP = 8,
```

Comment:
doesn't matter, but please align this as table by `=`

Reply:
Applied in #30661.